lib/core/schematics/migrations/collection.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
+  "schematics": {
+    "move-out-alfresco-api": {
+      "description": "Update alfresco-api imports",
+      "version": "7.0.0",
```

Comment:
Versions are going to be changed to current release version, so repos can update it like:

```
npx nx@17.0.2 migrate @alfresco/adf-core@6.10.0-9192302350
```